capstone/client/src/components/ListPage.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react';
+
+class ListPage extends Component {
+  render() {
+    return (
+      <div>Testing</div>
+    );
+  }
+}
+
+export default ListPage;
```

Comment:
Do not use default exports: https://google.github.io/styleguide/jsguide.html#es-module-exports